cmd/network/verification.go

```diff
@@ -22,25 +22,37 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const nonByovpcPrivateSubnetTagKey = "kubernetes.io/role/internal-elb"
+const (
+	nonByovpcPrivateSubnetTagKey = "kubernetes.io/role/internal-elb"
+	defaultInstanceType          = "t3.micro"
+	defaultTimeout               = 2
+)
 
-type egressVerification struct {
+type Egress struct {
 	awsClient egressVerificationAWSClient
 	cluster   *cmv1.Cluster
 	log       logging.Logger
 
-	// clusterId is the cluster identifier that will be used to query backplane for AWS credentials to build an AWS client
-	clusterId       string
-	region          string
-	subnetId        string
-	securityGroupId string
-	debug           bool
-	caCert          string
-	noTls           bool
+	// ClusterId is the cluster identifier that will be used to query backplane for AWS credentials to build an AWS client
+	ClusterId       string
+	Region          string
```

Comment:
looks to be parameters were previously marked as private, is there a reason for flipping this to public again?

Reply:
Ah yeah for this I tried to call it out in the description:

> Also makes the struct public so it can be used in cases like https://github.com/openshift/osdctl/pull/299 instead of having to duplicate code.

Sometimes other osdctl commands want to automatically run this afterwards, if it's public they can easily do this. Another approach would be to leave this private and create getter/setter functions

Reply:
ah ok, then let's keep it as is, I'd prefer using public than the getter/setter hell :)